Synapse/Api/Map.cs

```diff
@@ -129,6 +129,7 @@ public void SendBroadcast(ushort time, string message, bool instant = false)
             ScpRecontainmentType.Chaos => $". SCP {scp} SUCCESSFULLY TERMINATED . BY CHAOSINSURGENCY",
             ScpRecontainmentType.Scientist => $". SCP {scp} SUCCESSFULLY TERMINATED . BY SCIENCE PERSONNEL",
             ScpRecontainmentType.ClassD => $". SCP {scp} SUCCESSFULLY TERMINATED . BY CLASSD PERSONNEL",
+            ScpRecontainmentType.Scp => $"TERMINATED BY SCP {Unit}",
```

Comment:
This sounds like a direct communication to one specific player

However, this is for announcements across all players. Doesn't make much sense, nor do I understand the reasoning why you avoid the vanilla "Unknown"

Reply:
I don't understand

Reply:
it was the default annoncement